src/PlaywrightSharp/PlaywrightSharpException.cs

```diff
@@ -8,8 +8,15 @@ namespace PlaywrightSharp
     /// </summary>
     public class PlaywrightSharpException : Exception
     {
-        internal const string BrowserClosedExceptionMessage = "Browser has been closed";
-        internal const string BrowserOrContextClosedExceptionMessage = "Target page, context or browser has been closed";
+        /// <summary>
+        /// Message used when the browser gets closed.
+        /// </summary>
+        public const string BrowserClosedExceptionMessage = "Browser has been closed";
```

Comment:
Same as above (can we maybe have these consts somewhere else, would that make any sense?).

Reply:
I have two things in mind. A `Constants` class or a `CommonErrors` class. wdyt?

Reply:
So, I'd probably be inclined to move it to a `Resources` somewhere. Maybe not `resx` but... If we have strings that are actual resources, why not, right?

Reply:
I feel like resources are way too much overload for just a const string.

Reply:
Yeah, maybe too much. How about something like `CommonErrorMessages`, then?  And just put them in a folder/namespace `Resources`? How's that for a compromise?

Reply:
I like it! 👍